docs/dev/libs/cep.md

```diff
@@ -1373,7 +1372,7 @@ Pattern: `a b+` and sequence: `a b1 b2 b3` Then the results will be:
         <td>After found matching <code>a b1</code>, the match process will not discard any result.</td>
     </tr>
     <tr>
-        <td><strong>SKIP_TO_NEXT</strong>[<code>b*</code>]</td>
+        <td><strong>SKIP_TO_NEXT</strong>[<code>b</code>]</td>
```

Comment:
The pattern variable here is invalid at all. `SKIP_TO_NEXT` does not take any.

Reply:
Good Catch!